frontend/src/libs/components/sidebar-wrapper/libs/constants.ts

```diff
@@ -6,6 +6,7 @@ const SIDEBAR_ROUTES: Route[] = [
   { path: AppRoute.MEDITATION, name: 'meditation', icon: 'meditation' },
   { path: AppRoute.CHATS, name: 'chats', icon: 'chats' },
   { path: AppRoute.JOURNAL, name: 'journal', icon: 'journal' },
+  { path: AppRoute.PROFILE, name: 'profile', icon: 'avatar' },
```

Comment:
Initially, the 'avatar' icon was used solely for displaying the placeholder for user's avatar, will rename it into something more meaningful in next commits. Like 'User'